Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+
+# Build stage
+FROM rust:1.68 as builder
+
+RUN <<EOF
+    set -e
+
+    apt-get update
+    apt-get install -y build-essential
+
+    rm -rf /var/lib/apt/lists/*
+EOF
+
+WORKDIR /app
+
+COPY . .
+
+RUN --mount=type=cache,target=$RUSTUP_HOME,from=rust,source=$RUSTUP_HOME \
+    --mount=type=cache,target=$CARGO_HOME,from=rust,source=$CARGO_HOME \
+    --mount=type=cache,target=target \
+    cargo install --root output --path .
+
```

Comment:
We should also expose whatever port we use for the IPC endpoint. Here you have a sample template, we will have to handle them in tests so we can interact between images: https://github.com/consensus-shipyard/ipc-agent/blob/main/config/template.toml

Reply:
Fixed.